### PR TITLE
Explanation block and playground welcome page

### PR DESCRIPTION
**Pull request by vaivanov**

### CODE REVIEW

**Comment by jknnr** [comment]
In this version we haven't finished the footer. Now we have a content for it, so will make a new one soon!

**Comment by jknnr** [comment]
All good! Please check! @maxvisser

**Comment by dimitriirtimid** [comment]
@vaivanov I thing I forgot to mention. The build did not pass because you have lint warnings. You need to fix these before resubmitting. (You can easily check in the PR whether the builds run ok.

File: src/assets/jss/material-kit-react/components/explanationStyle.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import {
```
[the remark this answers is not in the document]
Comment (vaivanov) [reply src/assets/jss/material-kit-react/components/explanationStyle.jsx]: for WorkspaceWelcomeStyle and ExplanationStyle fixed in "Components styles paths" commit

File: src/components/Explanation/Explanation.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                            <img
+                                className={classes.image}
+                                src={require("assets/img/logo-kwf.png")}
+                                alt="Logo KWF Kankerfonds"
+                            />
+                        </a>
+                    </div>
+                    <div className={classes.right}>
+                        <p className={classes.text}>
+                            {t('Rookvrijspelen is een initiatief van Longfonds en ABN Ambro. Wij streven ernaar dat elk kind dat in 2019 wordt geboren Rookvrij kan opgroeien. Dat kan in een omgeving, zonder verleidingen en met goede voorbeelden om je heen. Hoe minder mensen je ziet roken, hoe minder vanzelfsprekend het wordt om zelf te gaan roken. Iedereen kan helpen met het Rookvrij maken van onze omgeving.')}
```
[the remark this answers is not in the document]
Comment (vaivanov) [reply src/components/Explanation/Explanation.jsx]: fixed in "text updates" commit

File: src/components/PlaygroundIcons/playgroundIconsConstants.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export const playgroundIcons = [{
+        bg: require('assets/img/texture-1@2x.png'),
+        icon: require('assets/img/icon-cooperate@2x.png'),
+        title: 'Voorbereiden',
+        text: '15 teamleden in uw buurt zijn bezig met het voorbereiden van het rookvrij maken van de lineushof. Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
```
[the remark this answers is not in the document]
Comment (vaivanov) [reply src/components/PlaygroundIcons/playgroundIconsConstants.js]: fixed in "text updates" commit

File: src/views/Workspace/Sections/WorkspaceWelcome.jsx [hunk 4]
```diff
@@ -113,99 +98,39 @@ class WorkspaceWelcome extends React.Component {
...
-                            { userIsVolunteer ? 
-                                <CollapseCard title={"Team"}
-                                            image={require("assets/img/backgrounds/team.jpg")}
-                                            content={contentMessage}
-                                            MoreInformation={"Meer informatie"}
-                                            ExpandContent={<PlaygroundManagers playground={playground} user={user}/>}
-                                />
+                <div className={classes.page}>
+                    <div className={classes.container}>
+                        <div className={classes.title}>Lineushof Rookvrij</div>
```
[the remark this answers is not in the document]
Comment (vaivanov) [reply src/views/Workspace/Sections/WorkspaceWelcome.jsx]: fixed in "text updates" commit

File: src/views/Workspace/Sections/WorkspaceWelcome.jsx [hunk 5]
```diff
@@ -113,99 +98,39 @@ class WorkspaceWelcome extends React.Component {
...
+                        <div className={classes.descr}>Deze actiepagina gaat over het rookvrij maken van de Lineushof. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed congue fringilla mi vel commodo. Integer a bibendum erat.</div>
```
[the remark this answers is not in the document]
Comment (vaivanov) [reply src/views/Workspace/Sections/WorkspaceWelcome.jsx]: fixed in "text updates" commit

File: src/views/Workspace/Sections/WorkspaceWelcome.jsx [hunk 6]
```diff
@@ -113,99 +98,39 @@ class WorkspaceWelcome extends React.Component {
...
-                                <CollapseCard title={"Team"}
-                                            image={require("assets/img/backgrounds/team.jpg")}
-                                            content={contentMessage}
-                                            primaryCta={"Word lid"}
-                                            primaryCtaOnClick={() => this.onJoinClicked()}
-                                            MoreInformation={"Meer informatie"}
-                                            ExpandContent={<PlaygroundManagers playground={playground} user={user}/>}
-                                />
+                                <a
+                                    className={classNames(classes.button, classes.passive)}
```
[the remark this answers is not in the document]
Comment (vaivanov) [reply src/views/Workspace/Sections/WorkspaceWelcome.jsx]: fixed in "build fixes and button styles"

File: src/assets/jss/material-kit-react/components/explanationStyle.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (vaivanov) [reply src/assets/jss/material-kit-react/components/explanationStyle.jsx]: sorry but i have no access to push in parent repository

[the remark this answers is not in the document]
Comment (dimitriirtimid) [reply src/assets/jss/material-kit-react/components/explanationStyle.jsx]: Ok for now. I discussed the approach for the future with Jamal

**Comment by dimitriirtimid** [comment]
copied the branch and merged into master

**Pull request closed without merging**